nifi-extension-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/db/ColumnDescription.java

```diff
@@ -84,9 +84,6 @@ public static ColumnDescription from(final ResultSet resultSet) throws SQLExcept
         return new ColumnDescription(columnName, dataType, required, colSize == 0 ? null : colSize, isNullable);
     }
 
-    public static String normalizeColumnName(final String colName, final boolean translateColumnNames) {
-        return colName == null ? null : (translateColumnNames ? colName.toUpperCase().replace("_", "") : colName);
```

Comment:
@mattyb149 GitHub shows you as the author of this line 88 back in 2021 when you created the ColumnDescription class. Do you feel that the null check is required here? 
Can you think of a use case where colName parameter would be null?
I ask because the replacement code does NOT have the null check and I want to see if you or others feel the null check is worth having in the new code?